CNAME

```diff
@@ -1 +0,0 @@
-shepherd.artpi.net
```

Comment:
Please dont remove this domain, otherwise it will stop working, since its hosted by github pages :)

Reply:
Okay, I re-added it. :)